api/lib/controllers/user.js

```diff
@@ -159,4 +160,58 @@ module.exports = {
 
     res.status(200).send(grps[0].groups);
   },
+
+  async actionItems(req, res) {
+    const userId = req.credentials.user._id;
+    let { completed, skip, limit } = req.query;
+
+    const filters = [];
+
+    if (completed) {
+      filters.push({ $match: { completed: JSON.parse(completed) } });
+    }
+
+    if (!skip) {
+      skip = 0;
+    } else {
+      skip = parseInt(skip);
+    }
```

Comment:
```suggestion
skip ? parseInt(skip) : 0
```

Just curious if you think this is more readable. Im always torn between doing it the two ways

Reply:
I find it easier to read with the separate blocks but I don't really think it matters either way unless the ternary takes up multiple lines or is nested in some way.